### PR TITLE
E-930 feat: add timeout for special clientIds

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: .env.development [hunk 1]
```diff
@@ -8,3 +8,5 @@ VUE_APP_GOOGLE_CLIENT_ID = 885568627115-2lu9inbk9n7o1hc0me6dip2ac5drprj0.apps.go
 VUE_APP_GIT_CLIENT_ID = 346108128ad51a5eccd1
 
 VUE_APP_KDF_PARAMS_N = 1024
+
+VUE_APP_EXTRA_TIMEOUT_FOR_CLIENT_IDS=['bf86799a-4d6d-4d7c-892a-e5559814aae1','1a23cc96-fcd8-4ed0-bdb3-038dd41e57e8']
```
Comment (lamartire) [inline .env.development:12]: Does `dotenv` correctly work with arrays?

File: src/components/modules/document/steps/extraLoading/ModeApp/ModeApp.interactor.vue [hunk 2]
```diff
@@ -34,6 +36,16 @@ export default {
     isRequiredDocsVerifiedStatus() {
       return this.$options.documentsRequiredStore.isRequiredDocsVerifiedStatus;
     },
+
+    timeout() {
```
Comment (lamartire) [inline src/components/modules/document/steps/extraLoading/ModeApp/ModeApp.interactor.vue:40]: Which `timeout`? For that?
Name has not context.

File: .env.development (hunk 1 above)
Comment (dudiq) [reply .env.development:12]: yes, it works

**Pull request merged**